### DISCUSSION TITLE
I need all my pwm pins

### DISCUSSION THREAD

**Discussion by noweare1** [Arduino-IRremote/Arduino-IRremote#979] [Q&A] [answered]
First off, thank you for this library. Very comprehensive and useful.
My question is that I am not using IR send and I need all my pwm pins
What is the easiest way to make the library not use the send part of
it so I can use pin 3.

**Comment by ArminJo** [comment, marked as answer]
Look at the examples, and you will be lucky 😀 .
For the ones stumbling in here: look at MinimalReceiver

**Reply by noweare1** [reply]
I am using an uno to develop and using the 6 pwm channels to drive mosfets to drive 12V tri-color leds. So after taking your suggestion and looking at the examples, well actually reading more, it looks like I did get lucky since my remote uses NEC protocol so maybe I can use
the timerless receiver code. I still may have some questions later. Thank you.

**Reply by ArminJo** [reply]
Glad you found it.
If you have suggestions for improvements even in documentation, you are welcome!
